app/src/main/java/com/example/dogbook/main/models/Post.java

```diff
@@ -23,6 +23,10 @@
     public static final String KEY_LOCATION = "location";
     public static final String KEY_AUTHOR = "author";
 
+    private boolean isLiked = false; //Default value: the post has not been liked by the user
```

Comment:
nit: `isLikedByViewer` or `isLikedByLoggedInUser` is something we'd probably use in app code